src/Frontend/Core.hs

```diff
@@ -125,6 +129,8 @@ type instance CaptureData User               = AUID User
 type instance CaptureData IdeaJuryResultType = IdeaJuryResultType
 type instance CaptureData Role               = Role
 
+data PostResult a = UnsafePostResult
```

Comment:
By unsafe I mean that there should not be more occurrences of it. The goal of this type is to make sure one call `runPostHandler` .